### PR TITLE
Add pymmcore-widgets recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
@tlambert03 do you agree with being added as maintainer?

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

**Comment by goanpeca** [comment]
This one is ready for review https://github.com/orgs/conda-forge/teams/staged-recipes :)

**Comment by tlambert03** [comment]
gentle nudge for review/merge please @conda-forge/help-python
thanks!

File: recipes/pymmcore-widgets/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - fonticon-materialdesignicons6
+    - pymmcore-plus >=0.6.3
+    - python >=3.8
+    - qtpy
+    - superqt >=0.3.1
+    - useq-schema >=0.1.2
```
Comment (jaimergp) [inline recipes/pymmcore-widgets/meta.yaml:28]:
```suggestion
    - useq-schema >=0.1.2
    - vispy
```

Do you want to add `vispy`? It's [needed](https://github.com/pymmcore-plus/pymmcore-widgets/blob/352b7e9a7bf92080d53ac8f09307a8a4f30a48a0/src/pymmcore_widgets/_image_widget.py#L41) in some modules, apparently.

Comment (jaimergp) [reply recipes/pymmcore-widgets/meta.yaml:28]:
```suggestion
    - useq-schema >=0.1.2
    - numpy
```

And [same](https://github.com/pymmcore-plus/pymmcore-widgets/blob/352b7e9a7bf92080d53ac8f09307a8a4f30a48a0/src/pymmcore_widgets/_image_widget.py#L12) for numpy.

Comment (tlambert03) [reply recipes/pymmcore-widgets/meta.yaml:28]: No thanks on both accounts.  Vispy is an extra needed for only one widget and a heavy enough dependency that id rather require users to have to conda install it explicitly.  And numpy is only needed for a type hint (for a getter function from a subdependency), it only gets imported inside of type checking blocks

**Comment by tlambert03** [comment]
thank you both @goanpeca and @jaimergp :)

**Pull request merged**